hrabsence/CRM/HRAbsence/Test/Fabricator/HRAbsencePeriod.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+class CRM_HRAbsence_Test_Fabricator_HRAbsencePeriod {
+
+  public static function fabricate($params = []) {
+    if(empty($params['title'])) {
```

Comment:
I guess it would be better to set a default data to the current year start and end data if the parameters are not set.